playground/components/header.tsx

```diff
@@ -62,90 +72,146 @@ export const Header: Component<{
         </a>
         <span class="inline-block -mb-1">Solid Playground</span>
       </h1>
-
       <div class="flex items-center space-x-2">
+        <div
+          classList={{ 'absolute top-[53px] right-[10px] w-[fit-content]': showMenu() }}
+          class="z-10"
+        >
+          <div
+            class="shadow-md md:items-center md:space-x-2 md:flex"
```

Comment:
We are missing a `shadow-none` on desktop here
```suggestion
            class="shadow-md md:shadow-none md:items-center md:space-x-2 md:flex"
```

Reply:
Moved shadow to classList and cleaned up this in general